packages/engine-ui/src/components/CreateBooking.js

```diff
@@ -38,18 +38,35 @@ const CreateBooking = ({ onSubmit }) => {
   const history = useHistory()
   const [isFinished, setIsFinished] = React.useState(false)
   const [formState, setState] = React.useState(initialState)
+  const [showErrorMessage, setErrorState] = React.useState({
```

Comment:
Since this is a getter and a setter I think the naming could better convey that relationship.
I think it's confusing to have the name `showErrorMessage`, which sounds like a boolean, and then it's in fact an object with properties.
```suggestion
  const [formErrors, setFormErrors] = React.useState({
```

Reply:
ifix